src/routes/index.svelte

```diff
@@ -102,5 +102,5 @@
     {/each}
   </main>
 {:else}
-  <main class="text-center">Unfortuately there was no issue found</main>
+  <main class="text-center">Unfortunately, there was no issues found.</main>
```

Comment:
I'm not a native speaker but imo this sounds better:

```suggestion
  <main class="text-center">Unfortunately, there were no issues found.</main>
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question